keyboards/meletrix/zoom98/info.json

```diff
@@ -0,0 +1,215 @@
+{
+  "keyboard_name": "zoom98",
+  "manufacturer": "meletrix",
+  "url": "https://meletrix.com/",
+  "maintainer": "spbgzh",
+  "usb": {
+    "vid": "0x36B5",
```

Comment:
While this vendor ID isn't in use, at this exact moment, 0x36A9 is the last assigned one, currently.  This will be assigned in the very, very near future. 

Could you use a different vendor ID?  The further past 0x4000, the better.

Reply:
we signed this ID
![image](https://github.com/qmk/qmk_firmware/assets/72414103/829e2dba-d29b-4f0e-86b5-0a3852f7f406)

Reply:
Thank you for the verification.  
And it sounds like this is super new, as I did check the actual USB-IF list, which was updated August 2nd.